docs/howto/operator.rst

```diff
@@ -101,3 +101,80 @@ to execute a BigQuery load job.
     :dedent: 4
     :start-after: [START howto_operator_gcs_to_bq]
     :end-before: [END howto_operator_gcs_to_bq]
+
+GcfFunctionDeleteOperator
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the ``default_args`` dict to pass arguments to the operator.
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcf_function_delete_dag.py
+    :language: python
+    :start-after: [START howto_operator_gcf_delete_args]
+    :end-before: [END howto_operator_gcf_delete_args]
+
+
+Use the :class:`~airflow.contrib.operators.gcp_function_operator.GcfFunctionDeleteOperator`
+to delete a function from Google Cloud Functions.
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcf_function_delete_dag.py
+    :language: python
+    :start-after: [START howto_operator_gcf_delete]
+    :end-before: [END howto_operator_gcf_delete]
+
+
+GcfFunctionDeployOperator
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the :class:`~airflow.contrib.operators.gcp_function_operator.GcfFunctionDeployOperator`
+to deploy a function from Google Cloud Functions.
+
+The examples below use Airflow variables defined in order to show various variants and
+combinations of default_args you can use. The variables are defined as follows:
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcf_function_deploy_delete.py
+    :language: python
+    :start-after: [START howto_operator_gcf_deploy_variables]
+    :end-before: [END howto_operator_gcf_deploy_variables]
+
+With those variables one can define body of the request:
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcf_function_deploy_delete.py
+    :language: python
+    :start-after: [START howto_operator_gcf_deploy_body]
+    :end-before: [END howto_operator_gcf_deploy_body]
+
+The default_args dictionary when you create DAG can be used to pass body and other
+arguments:
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcf_function_deploy_delete.py
+    :language: python
+    :start-after: [START howto_operator_gcf_deploy_args]
+    :end-before: [END howto_operator_gcf_deploy_args]
+
+
+Note that the body nor default args are not yet complete in the above examples.
```

Comment:
Note that the neither the body nor default args are complete in the above examples.

Reply:
Ah yeah. Thanks! Will have to make some automated grammar check next time :).

Reply:
BTW. You might be surprised but in Polish double negative exists and is just stronger negative: https://en.wikipedia.org/wiki/Double_negative